lib/install.js

```diff
@@ -1,43 +1,45 @@
 'use strict';
 
 var fs = require('fs');
-var mkdirp = require('mkdirp');
 var path = require('path');
 var async = require('async');
+var rimraf = require('rimraf');
 var cli = require('./cli');
 var config = require('./config');
-var driverUtils = require('./driver-utils');
+var initDriver = require('./init-driver');
 var configureIe = require('./configure-ie');
 
-Object.keys(config.binaries).forEach(function (resource) {
-  mkdirp.sync(config.binaries[resource].binary.path);
-});
+try {
+  // reset drivers if any critical config has changed (such as arch version).
```

Comment:
I don't think arch changes would normally happen right?  I mean, you'd re-install your OS if that occurred.